### ISSUE TITLE
feature request: option to log server adrress link 

### ISSUE THREAD

**Issue by reslear** [axe-me/vite-plugin-node#1]
like: http://localhost:3000/

**Comment by reslear** [comment]
```ts
console.log(chalk.green.underline('https://localhost:3000') )
```

**Comment by reslear** [comment]
or add hook before launching

**Comment by axe-me** [comment]
@reslear good suggestion, in fact, the configured port is printed in debug mode. I can move it to regular log.

btw, just curious how did you find this plugin, I just published it 1 day ago and not post it anywhere yet.

**Comment by reslear** [comment]
Google -> vite express server -> npm -> GitHub

**Comment by axe-me** [comment]
shipped in 0.0.6
